view/mammography_patient_tree.xml

```diff
@@ -7,7 +7,9 @@ contains the full copyright notices and license terms. -->
         <tree>
             <field name="name" expand="1" widget="char"/>
             <field name="dob" expand="1"/>
-            <field name="puid" expand="1"/>
+            <field name="partner_patient_id" expand="1"/>
+            <field name="mobile" expand="1"/>
+            <field name="most_recent_imaging_request_datetime" expand="1" widget="date"/>
```

Comment:
part 2, 3, 4 removes PUID, and adds patient_id and mobile to the tree view.

You might notice I added the `most_recent_imaging_request_datetime` value to, but I wasn't able to make it searchable yet. Will look into that as a follow up when it's prioritised.